annotations/views/amphora_views.py

```diff
@@ -115,8 +115,8 @@ def retrieve(self, request, pk=None, repository_pk=None):
         aggregate_content = result.get('aggregate_content')
 
         submitted = False
-        for child in range(len(master_text.children)):
-            if Appellation.objects.filter(occursIn_id=master_text.children[child], submitted=True):
+        for index, child in enumerate(master_text.children):
```

Comment:
if you are not using the index why do you use enumerate here? Why not just loop over the text?